suggestions & corrections

vordimous opened the pull request.

vordimous commented on README.md:
```diff
@@ -15,51 +15,57 @@
 
-##  A multi-protocol edge and service proxy, designed for event-driven architectures
-Zilla abstracts Apache Kafka® for web applications, IoT clients and non-Kafka microservices. With Zilla, Kafka topics can be securely and reliably exposed via a user-defined `REST`, `Server-Sent Events (SSE)`, `MQTT`, or `gRPC` API.
+Zilla abstracts Apache Kafka® for web applications, IoT clients, and non-Kafka microservices. With Zilla, Kafka topics can be securely and reliably exposed via a user-defined `REST`, `Server-Sent Events (SSE)`, `MQTT`, or `gRPC` API.
```
```
Zilla abstracts Apache Kafka® for web applications, IoT clients, and non-Kafka microservices.
```

I'm not sure what a `non-Kafka microservice` is. My guess is microservices that don't connect to Kafka through an SDK or API. Maybe we can just say `microservice` or try to clarify it some more?

vordimous commented on README.md:
```diff
@@ -15,51 +15,57 @@
+Zilla abstracts Apache Kafka® for web applications, IoT clients, and non-Kafka microservices. Zilla can securely and reliably expose Kafka topics via a user-defined `REST`, `Server-Sent Events (SSE)`, `MQTT`, and `gRPC` APIs.
 
-Zilla has no external dependencies and does not rely on the Kafka Consumer/Producer API or Kafka Connect. Instead, it natively supports the Kafka wire protocol and uses advanced protocol mediation to establish stateless API entry points into Kafka. Besides protocol meditation, Zilla addresses security enforcement, observability and connection offloading on the data path.
+Zilla has no external dependencies and does not rely on the Kafka Consumer/Producer API or Kafka Connect. Instead, it natively supports the Kafka wire protocol and uses advanced protocol mediation to establish stateless API entry points into Kafka. Zilla also addresses security enforcement, observability, and connection offloading on the data path.
```
```
 connection offloading on the data path.
```
I don't understand what this means. The closest thing I could find was [TCP connection offloading](https://en.wikipedia.org/wiki/TCP_offload_engine).

llukyanov replied:
The number of connections from clients into Zilla does not have to equal the number of connections coming out of Zilla into Kafka , i.e. fan-in and fan-out.

vordimous replied:
Ok that makes more sense. If we can find where this terminology is documented, I am good with it. Otherwise, we may want to figure out how to reword it. Particularly since that is a huge feature and part of the first few sentences

vordimous commented on README.md:
```diff
@@ -15,51 +15,57 @@
 
-##  A multi-protocol edge and service proxy, designed for event-driven architectures
-Zilla abstracts Apache Kafka® for web applications, IoT clients and non-Kafka microservices. With Zilla, Kafka topics can be securely and reliably exposed via a user-defined `REST`, `Server-Sent Events (SSE)`, `MQTT`, or `gRPC` API.
+Zilla abstracts Apache Kafka® for web applications, IoT clients, and non-Kafka microservices. Zilla can securely and reliably expose Kafka topics via a user-defined `REST`, `Server-Sent Events (SSE)`, `MQTT`, and `gRPC` APIs.
```
```
Zilla abstracts Apache Kafka® for web applications, IoT clients, and non-Kafka microservices.
```

I'm not sure what a `non-Kafka microservice` is. My guess is microservices that don't connect to Kafka through an SDK or API. Maybe we can just say `microservice` or try to clarify it some more?

llukyanov replied:
Agreed, it should just be `microservice`.

vordimous commented on README.md:
```diff
@@ -15,51 +15,57 @@
 ### REST-Kafka Proxying
+
 - [x] **Correlated Request-Response (sync)** —  `HTTP` request-response over a pair of Kafka topics, correlated. Supports synchronous interaction, blocked waiting for a correlated response. 
 - [x] **Correlated Request-Response (async)** — `HTTP` request-response over a pair of Kafka topics, correlated. Supports asynchronous interaction, returning immediately with `202`. Accepted plus location to retrieve correlated response. Supports `prefer: wait=N` to retrieve the correlated response immediately as soon as it becomes available, with no need for client polling.
```
```
`HTTP` request-response over a pair of Kafka topics, correlated.
```
This sentence in both places feels a bit robotic and like reference documentation. Maybe we can rewrite these to be more reader friendly.

vordimous replied:
Or if this information is supposed to be dry, then a table or something to format it more uniformly.

llukyanov replied:
I tried using a table format, but it gets too messy and looks too "heavy." Friendlier wording is the better approach.

vordimous replied:
Ok cool. We can just take this as a task and leave this as is for now.

vordimous replied:
https://github.com/aklivity/aklivity-docs/issues/26

vordimous commented on README.md:
```diff
@@ -122,40 +133,41 @@ BufferBM.single           thrpt   15  15111915.264 ± 294689.110  ops/s
 
 This benchmark was executed on 2019 MacBook Pro laptop with `2.3 GHZ 8-Core Intel i9 chip` and `16 GB of DDR4 RAM`, showing about `14-15 million messages per second`.
```
We should probably do this on a standard EC2 instance vs a MBP

llukyanov replied:
Agreed!

vordimous replied:
cool Ill add this as another task https://github.com/aklivity/aklivity-docs/issues/27

llukyanov committed:
Merge pull request #305 from llukyanov/develop

Readme overhaul

llukyanov committed:
Update README.md

vordimous committed:
develop

- add feedback
- no numbers
- add bold back
- wording
- clean up some formattion and wording
- suggestions & corrections

llukyanov committed:
Update README.md

llukyanov committed:
Update README.md

llukyanov committed:
Update README.md
